save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/AgentService.kt

```diff
@@ -0,0 +1,29 @@
+package org.cqfn.save.orchestrator.service
+
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestTemplate
+import org.springframework.web.reactive.function.client.WebClient
+
+@Service
+class AgentService {
+    /**
+     * Used to send requests to backend
+     */
+    private val webClient = WebClient.create("http://localhost:5000")
```

Comment:
Oh, also, backend will be in another docker container, so it's host won't be localhost. It should be extracted to properties, localhost sohuld be used for `dev` profile and hostname from docker compose in prod profile

Reply:
Like [here](https://docs.docker.com/compose/networking/), another container should be reachable by it's name, as defined in docker-copose.yaml.